package.json

```diff
@@ -11,8 +11,6 @@
   "dependencies": {
     "@portabletext/react": "^3.0.4",
     "@sanity/client": "^6.6.0",
-    "@types/node": "20.8.2",
-    "@types/react": "18.2.14",
     "@types/react-dom": "18.2.6",
```

Comment:
Why this change?

Reply:
I am not sure about this change, I didn't  manually changed it myself.